### DISCUSSION TITLE
menu bar of smplayer dissapeared.

### DISCUSSION THREAD

**Discussion by andrew-ysk** [smplayer-dev/smplayer#296] [General]
I am trying to learn to get use of using smplayer ..

Where can i find the complete list of default keyboard shortcut for it ? 

My menu bar suddenly disappeared.. how to get it back ? 
I am using smplayer 21.8.0-1

Thx

**Comment by smplayer-dev** [comment]
In preferences -> keyboard and mouse you can see all shortcuts.

**Reply by andrew-ysk** [reply]
I have being digging thru all the shortcuts in ctrl-p   > keyboard & mouse , but there is None that i need, which is to make floating control bar appears during full screen. (instead of moving mouse to make it appear, i want to use keyboard shortcut instead). 

2. There is no command to make smplayer menu bar reappear. I am using "mini gui" in GUI option under "interface"

**Reply by smplayer-dev** [reply]
Maybe you enabled compact mode. Press Ctrl+C to turn it off.

**Reply by andrew-ysk** [reply]
Ctrl-C is the magic key.. Thank you.

**Reply by andrew-ysk** [reply]
How about floating control bar in full screen mode ?  Is there a keyboard cmd that makes the floating control bar appears ?

**Reply by smplayer-dev** [reply]
I'm afraid there isn't.

**Reply by andrew-ysk** [reply]
ok. Thanks for letting me know.
